Common/Securities/SecurityManager.cs

```diff
@@ -327,9 +328,18 @@ protected virtual void OnCollectionChanged(NotifyCollectionChangedEventArgs chan
             // add the symbol to our cache
             if (addToSymbolCache) SymbolCache.Set(symbol.Value, symbol);
 
-            //Add the symbol to Data Manager -- generate unified data streams for algorithm events
-            var config = subscriptionManager.Add(factoryType, symbol, resolution, dataTimeZone, exchangeHours.TimeZone, isCustomData, fillDataForward,
-                extendedMarketHours, isInternalFeed, isFilteredSubscription);
+            // Add the symbol to Data Manager -- generate unified data streams for algorithm events
+            SubscriptionDataConfigList configList = new SubscriptionDataConfigList(symbol);
+
+            // Get the type that will be used in the data feed
+            // Could be more than one for a given security - i.e. More than one subscription needed
+            foreach (var dataFeed in availableDataFeeds[symbol.ID.SecurityType])
+            {
+                var dataFeedType = GetDataFeedType(factoryType, dataFeed);
```

Comment:
Reusing the term data feed even makes me uneasy. Read up on the concept of ubiquitous language. 

There can only be on definition of data feed on the system and it's already been taken.